backend/satellite_tools/repo_plugins/yum_src.py

```diff
@@ -930,4 +930,4 @@ def _fix_encoding(text):
     if isinstance(text, str):
         return str.encode(text, 'utf-8')
     else:
-        return str(text)
+        return str(re.sub(u"(\u2018|\u2019)", "", text))
```

Comment:
This wrong. Why is that `‘` (`\u2018`) and `’` (`\u2019`) are unprintable? They aren't ASCII at first place and perfectly working with UTF-8 both ways:

```
print("\u2018\u2019".encode("utf-8"))
b'\xe2\x80\x98\xe2\x80\x99'
```

Second, if you want to "fix encoding" to just remove these symbols away and stay compatible with the ASCII, then you should just encode the string:

```python
printable = "\u2018\u2019".encode("ascii", errors="ignore")
```

In this case `printable` will be `b''`.

Reply:
why? i had issue, quick fix ensued, assumed someone else had same issue, done PR
I will wait for new code anyway (new uyuni release) this PR is more of legacy issue resolving rather than real code fix

Reply:
@bmanojlovic no problem with the issue fix. :-) Problem with the fact that in any RPM package _anything_ can happen as anyone can put anything in description or summary fields of the spec.  So the right way to get rid of those non-ASCII symbols would  be just as I showed in the next example:

```python
clean = broken.encode("ascii", errors="ignore")
```
And as I understand, that happens on Python 2.7 environment.

Reply:
changed it to .encode now